src/Symfony/Component/Security/composer.json

```diff
@@ -37,17 +37,12 @@
         "symfony/finder": "~2.3|~3.0.0",
         "symfony/polyfill-intl-icu": "~1.0",
         "symfony/routing": "~2.2|~3.0.0",
-        "symfony/translation": "~2.0,>=2.0.5|~3.0.0",
         "symfony/validator": "~2.5,>=2.5.5|~3.0.0",
-        "doctrine/common": "~2.2",
-        "doctrine/dbal": "~2.2",
-        "psr/log": "~1.0",
         "symfony/expression-language": "~2.6|~3.0.0",
-        "symfony/ldap": "~2.8|~3.0.0"
+        "symfony/ldap": "~2.8|~3.0.0",
+        "psr/log": "~1.0"
```

Comment:
oh, and it's already at the last post in master:
https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Security/composer.json#L44